classes/stock/StockManagerInterface.php

```diff
@@ -52,30 +51,29 @@ public static function isAvailable();
      *
      * @return bool
      */
-    public function addProduct($id_product, $id_product_attribute, Warehouse $warehouse, $quantity, $id_stock_movement_reason, $price_te, $is_usable = true, $id_supply_order = null);
+    public function addProduct($id_product, $id_product_attribute, $quantity, $id_stock_movement_reason, $price_te, $is_usable = true, $id_supply_order = null);
```

Comment:
Note: This may break a lot of modules because when upgrading from PS 1.7 to PS 8.0, the `$warehouse` parameter will be considered the `$quantity` parameter.

Reply:
Yeah, and this param was not deprecated 🤔